recipe/build.sh

```diff
@@ -1,19 +1,20 @@
 #!/bin/bash
 
-if [[ "${cuda_compiler_version}" =~ 12.* ]]; then
-  export CUDA_HOME=${BUILD_PREFIX}
-elif [[ "${cuda_compiler_version}" != "None" ]]; then
-  export CUDA_HOME=${CUDA_PATH}
+if [[ -z "${CUDA_HOME+x}" ]]; then
+  # `$CUDA_HOME` was set for CUDA 11.x and earlier.
+  # Must be using CUDA 12.0+. So set to `$BUILD_PREFIX`.
+  # This is needed to find `nvcc` and `cudart`.
+  export CUDA_HOME="${BUILD_PREFIX}"
 fi
```

Comment:
@danpetry have simplified this a bit as `CUDA_HOME` is already set for CUDA 11.x (and earlier). So we can just check if `CUDA_HOME` is set and if not handle the CUDA 12.0+ case

Please let me know if you have thoughts on this approach

Reply:
That looks good to me